SDK Showcase/SDK Showcase/Interactors/ResourceInteractor.swift

```diff
@@ -4,44 +4,75 @@ import OneginiSDKiOS
 import SwiftUI
 
 protocol ResourceInteractor {
-    func fetchDeviceList()
-    func fetchImplicit()
+    /// Requests performed on UserClient
+    func sendAuthenticatedRequest()
+    func sendImplicitRequest()
+    
+    /// Requests performed on UserDevice
+    func sendUnauthenticatedRequest()
+    func sendAnonymousRequest()
 }
 
 class ResourceInteractorReal: ResourceInteractor {
     private let userClient = SharedUserClient.instance
+    private let deviceClient = SharedDeviceClient.instance
     private let decoder = JSONDecoder()
     @ObservedObject var app: ShowcaseApp
 
     init(app: ShowcaseApp) {
         self.app = app
     }
     
-    func fetchDeviceList() {
+    func sendUnauthenticatedRequest() {
+        // Should be defined on the access
+        let pathToTheResource = "path-to-the-resource"
```

Comment:
Magic "number"? Is if final path? :)

Reply:
no it is not final, this only show the developer how to deal with the api. We do not have resources defined for this kind of auth